examples/platform/linux/AppMain.cpp

```diff
@@ -65,6 +65,9 @@ using namespace chip::Transport;
 using chip::Shell::Engine;
 #endif
 
+static constexpr useconds_t kWifiStartCheckTime  = 100 * 1000; // 100 ms
```

Comment:
please name this `kWifiStartCheckTimeMs` or similar, to make the unit of measurement very clear